src/classes/RD2_Constants.cls

```diff
@@ -80,6 +80,12 @@ public inherited sharing class RD2_Constants {
     * @description Recurring Donation "Recurring Type" picklist API values
     */
     public static final String RECURRING_TYPE_OPEN = 'Open';
+    public static final String RECURRING_TYPE_FIXED = 'Fixed';
```

Comment:
[RD Health check](https://github.com/SalesforceFoundation/NPSP/blob/master/src/classes/STG_PanelRDHealthCheck.cls#L399) supported picklist values need to be modified to account for the fixed recurring type. 

@rponti-sforg , this is related to PR #5060 testing ensuring there is no health check error result now if there are RDs with Recurring Type = Fixed.

Reply:
Thanks for the reminder @RadSFDeveloper